docs/src/index.md

```diff
@@ -26,6 +26,29 @@ in Julia.
     Your changes will be vetted by developers before becoming permanent, so don't
     worry about whether you might say something wrong.
 
-```@contents
-Pages = ["install.md", "quickstart.md", "arrays_colors.md", "conversions_views.md", "indexing.md", "imageaxes.md", "imagefiltering.md", "imagemetadata.md", "imagesegmentation.md", "function_reference.md", "api_comparison.md"]
+This documentation is a collection of two parts: a top-level documentation that guides you into the
```

Comment:
Would it be better to say "This documentation is a collection of several parts." and then delete the next couple of sentences, and launch straight into 'The "Tutorials" part...'?